tests/unit/s2n_tls13_cert_verify_test.c

```diff
@@ -46,12 +48,12 @@ int run_tests(const struct s2n_tls13_cert_verify_test *test_case, s2n_mode verif
     const char *key_file = test_case->key_file;
     struct s2n_signature_scheme sig_scheme = *test_case->sig_scheme;
 
-    struct s2n_config *config = NULL;
-    EXPECT_NOT_NULL(config = s2n_config_new());
```

Comment:
Since every test in this scope uses the same certificate (test_case->cert_file), can you just add the certificate to this config once at the beginning?